solver/llbsolver/solver.go

```diff
@@ -241,10 +241,10 @@ func (s *Solver) Solve(ctx context.Context, id string, sessionID string, req fro
 			}
 			inp.Refs = m
 		}
-		if _, ok := asInlineCache(exp.CacheExporter); ok {
+		if _, ok := asInlineCache(out.CacheExporter); ok {
```

Comment:
It doesn't look like these parts should run for each exporter. Only the actual `Export()` call iiuc.